src/dynamodb/batch/write.js

```diff
@@ -26,6 +27,8 @@ function ensureValidParameters(documentClient, tableName, items) {
   if (!documentClient) throw new Error('documentClient is required.');
   if (!tableName) throw new Error('Table name is required.');
   if (!items) throw new Error('Item list is required.');
+  if (isMultidimensional(items))
+    throw new Error("Item list can't be multidimensional.");
```

Comment:
I think the naming with "multidimensional" is good but might consider replacing it here with something like "item list can't contain arrays" or at least add something like that for clarity